From slony1-bugs at lists.slony.info  Sun Dec  6 16:51:10 2009
From: slony1-bugs at lists.slony.info (slony1-bugs@lists.slony.info)
Date: Sun Dec  6 16:51:12 2009
Subject: [Slony1-bugs] [Bug 67] Array overrun in logtrigger()
In-Reply-To: <bug-67-4@http.www.slony.info/bugzilla/>
Message-ID: <20091207005110.436DC290C85@main.slony.info>

http://www.slony.info/bugzilla/show_bug.cgi?id=67


Devrim GUNDUZ <devrim@commandprompt.com> changed:

           What    |Removed                     |Added
----------------------------------------------------------------------------
                 CC|                            |devrim@commandprompt.com




-- 
Configure bugmail: http://www.slony.info/bugzilla/userprefs.cgi?tab=email
------- You are receiving this mail because: -------
You are on the CC list for the bug.
You are the assignee for the bug.
From slony1-bugs at lists.slony.info  Sun Dec  6 16:59:37 2009
From: slony1-bugs at lists.slony.info (slony1-bugs@lists.slony.info)
Date: Sun Dec  6 16:59:38 2009
Subject: [Slony1-bugs] [Bug 67] Array overrun in logtrigger()
In-Reply-To: <bug-67-4@http.www.slony.info/bugzilla/>
Message-ID: <20091207005937.078E8290C3D@main.slony.info>

http://www.slony.info/bugzilla/show_bug.cgi?id=67





--- Comment #7 from Devrim GUNDUZ <devrim@commandprompt.com>  2009-12-06 16:59:36 ---
Was this ever fixed? We got a report from a customer:

org.postgresql.util.PSQLException: ERROR: Slony-I: old key column xxx.tablename
IS NULL on UPDATE 


This is a table that has 27 columns, but:

"_CLUSTER".logtrigger('_CLUSTER', '1221', 'kvvvvvvvvvvvvvvvvvvvvvv')

What is the correct fix here?

D.


-- 
Configure bugmail: http://www.slony.info/bugzilla/userprefs.cgi?tab=email
------- You are receiving this mail because: -------
You are on the CC list for the bug.
You are the assignee for the bug.
From slony1-bugs at lists.slony.info  Fri Dec 11 02:38:34 2009
From: slony1-bugs at lists.slony.info (slony1-bugs@lists.slony.info)
Date: Fri Dec 11 02:38:37 2009
Subject: [Slony1-bugs] [Bug 104] New: slon core in sched_add_fdset
Message-ID: <bug-104-4@http.www.slony.info/bugzilla/>

http://www.slony.info/bugzilla/show_bug.cgi?id=104

           Summary: slon core in sched_add_fdset
           Product: Slony-I
           Version: 1.2
          Platform: PC
        OS/Version: Linux
            Status: NEW
          Severity: normal
          Priority: medium
         Component: slon
        AssignedTo: slony1-bugs@lists.slony.info
        ReportedBy: harald.kolb@nsn.com
                CC: slony1-bugs@lists.slony.info
   Estimated Hours: 0.0


We sometimes get a core of slon process when stopping it (via pkill slon).
We are using slony1-2.0.3-rc2 on RedHat Linux on x86 hardware.

Pstack is:
Program terminated with signal 11, Segmentation fault.
#0  0x0805dcc7 in sched_add_fdset (fd=-1, fds=0x806e780) at scheduler.c:681
681     scheduler.c: No such file or directory.
        in scheduler.c
(gdb) bt
#0  0x0805dcc7 in sched_add_fdset (fd=-1, fds=0x806e780) at scheduler.c:681
#1  0x0805e0a9 in sched_wait_conn (conn=0xb4a00490, condition=5)
    at scheduler.c:211
#2  0x0804f45d in remoteListenThread_main (cdata=0x80ad4d0)
    at remote_listen.c:338
#3  0xb7ebde49 in start_thread () from /lib/libpthread.so.0
#4  0xb7e4ccce in clone () from /lib/libc.so.6
(gdb)


-- 
Configure bugmail: http://www.slony.info/bugzilla/userprefs.cgi?tab=email
------- You are receiving this mail because: -------
You are on the CC list for the bug.
You are the assignee for the bug.
